### PR TITLE
Fix uni-primary-button rules

### PR DESCRIPTION
**Pull request by PatrykSzwer**
Right now uni-primary-button doesn't look consistent with other button selectors. I've removed unnecessary styles and fixed hover/focus which didn't work. 

Also there is one thing to discuss. According to our [design](https://user-images.githubusercontent.com/2656996/34675878-356a6f6e-f48b-11e7-89a3-a5bd85a3c845.png), buttons should have brighter color when they are hovered (and then darker when clicked down) but right now uniform works quite opposite (button is darker when you hover it). For me it looks fine how it is now, but maybe we should go with exact same styles as in design. WDYT?

### CODE REVIEW

File: src/uniform.css [hunk 1]
```diff
@@ -67,28 +67,22 @@
...
    background-color: var(--primary) !important;
-   border: 0.1rem solid var(--primary) !important;
-   border-radius: 3px !important;
-   height: inherit !important;
  }
 
  .uni-primary-button-wrapper > ::slotted(button:hover),
  .uni-primary-button-wrapper > ::slotted(button:focus) {
-   background-color: var(--primary) !important;
+   filter: brightness(90%) !important;
```
Comment (warpech) [inline src/uniform.css]: This is not configurable. I would rather do it through the configruation option that is compatible with https://github.com/Starcounter/underwear.css/blob/c595b74a49e6e538af4bcba6f1c9c64289566fb2/underwear.unminified.css#L561-L567

File: src/uniform.css [hunk 2]
```diff
@@ -67,28 +67,22 @@
...
  }
  .uni-primary-button-wrapper > button:hover,
  .uni-primary-button-wrapper > button:focus {
-   background-color: var(--primary) !important;
+   filter: brightness(90%) !important;
```
Comment (warpech) [inline src/uniform.css]: Same as above.

**Comment by warpech** [comment]
@tomalec can you please take over this PR so that @PatrykSzwer can continue being focused on BlendingEditor itself.

**Comment by tomalec** [comment]
I'll unify sizing of buttons, but for the colors I'd wait for https://github.com/Starcounter/uniform.css/issues/6#issuecomment-364880229 and fix it there.

**Comment by PatrykSzwer** [comment]
I found one more thing. "Disabled" functionality for buttons is missing. Right now there is no difference between disabled success button and enabled one. Could You please add it in this PR @tomalec ?

**Comment by tomalec** [comment]
> there is **no** difference between disabled success button and enabled one.

I agree the difference is not as in design. But to have **any** difference, even with not-implemented-yet `uniform.css`  make sure you use `underwear.css`

**Comment by warpech** [comment]
> I agree the difference is not as in design. But to have any difference, even with not-implemented-yet uniform.css make sure you use underwear.css

I disagree... We have agreed that Uniform should work with or without Underwear.

I agree with @PatrykSzwer that it would be great to add "disabled" state of primary, etc buttons to the demo page.

**Comment by tomalec** [comment]
> I agree with @PatrykSzwer that it would be great to add "disabled" state of primary, etc buttons to the demo page.

There is disabled state added to the demo, there is no disabled state added to the `uniform.css`.
I totally agree we should have disabled states in the demo as we do right now. 

I agree uniform should work with and without underwear. 

The question is whether it should look exactly the same with and without. In my opinion, it doesn't have to. That why we separated them, not included on into another.


Maybe I'd start to write a clear list of things we agreed on, to be implemented for buttons
- [ ] disabled state,
- [ ] common sizing and margins,
- [ ] colors that fit the design
   - [ ] consult with @andwah whether we should have separate color for each state, or let browser calculate it (`brightness, opacity, filters, calc`)

**Pull request merged**